index.html

```diff
@@ -1,188 +1,202 @@
+                <a href="#"><img class="logo" alt="Logo icon" src="img\Logo Placeholder.svg"></a>
                 <nav class="menu">
                     <ul>
-                        <li><p>Portfolio</p></li>
-                        <li><p>About</p></li>
-                        <li><p>Contact</p></li>
+                        <li>
+                            <p>Portfolio</p>
+                        </li>
+                        <li>
+                            <p>About</p>
+                        </li>
+                        <li>
+                            <p>Contact</p>
+                        </li>
                     </ul>
-                    <img class="bar" src="img\Icon - Menu.svg" alt="hambuger bar menu">
+                    <img class="bar" src="img\Icon - Menu.svg" alt="hambuger icon">
                 </nav>
             </header>
             <section class="content">
                 <h1>I’m Shinhyo<br> Glad to see you!</h1>
                 <p>
-                    I’m a software developer! I can help you build a product , 
-                    feature or website Look through some of my work and experience! 
-                    If you like what you see and have a project you need coded, 
-                    don’t hestiate to contact me.
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded,
+                    don’t hesitate to contact me.
                 </p>
                 <h3 class="social">LET’S CONNECT</h3>
                 <ul class="socialIcon">
                     <li><a href="#"><img class="icon" src="img\Vector.svg" alt="Tweeter icon"></a></li>
                     <li><a href="#"><img class="icon" src="img\Linkedin icon.svg" alt="Linkedin icon"></a></li>
-                    <li><a href="#"><img class="icon" src="img\Vector m.svg" alt="M icon"></a></li>
-                    <li><a href="#"><img class="icon" src="img\github icon.svg" alt="github icon"></a></li>
-                    <li><a href="#"><img class="icon" src="img\peace icon.svg" alt="peace icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\Vector m.svg" alt="Mail icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\github icon.svg" alt="Github icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\peace icon.svg" alt="Peace icon"></a></li>
                 </ul>
             </section>
         </section>
         <section id="work">
-            <!-- projecto 1 -->
+            <!-- project 1 -->
             <section class="project">
-                <img class="snap" src="img\Snapshoot Portfolio.svg" alt="Project picture">
+                <img class="snap" src="img\Snapshoot Portfolio.svg" alt="First project card picture">
                 <h2>Tonic</h1>
```

Comment:
- [ ] The page has at least one heading `<h1>`, and here you use two different `hx` in one tag please kindly fix that 🙃. `eg: <h2>Tonic</h2>` 👍 👍